dev/github/Dockerfile-jdk11

```diff
@@ -11,7 +11,22 @@
 
 # See https://hub.docker.com/r/alluxio/alluxio-maven for instructions on running the image.
 
+ARG RUST_VERSION=1.66.1
+FROM rust:${RUST_VERSION} as rust-installer
+RUN rustup target add aarch64-unknown-linux-gnu x86_64-unknown-linux-gnu && \
+    rustup component add clippy rustfmt && \
+    # move the two directories into an empty directory to save one COPY instruction in the next stage
+    mkdir /cargo-and-rustup && \
+    mv /usr/local/rustup /cargo-and-rustup/rustup && \
+    mv /usr/local/cargo /cargo-and-rustup/cargo
+
 FROM maven:3.8.6-jdk-11
+ARG RUST_VERSION
```

Comment:
why is RUST_VERSION declared as an ARG twice?

Reply:
ARGs only last for the build phase of a single image. See
https://stackoverflow.com/questions/53681522/share-variable-in-multi-stage-dockerfile-arg-before-from-not-substituted